app/vodic/vodic-responsive.css

```
@@ -0,0 +1,254 @@
+.mob_secton_four .mb_23px{ margin-bottom: 23px !important; }
+
+.mob_secton_four .mt_93px{ margin-top: 93px !important; }
+.mob_secton_four .mt_20px{  margin-top: 20px !important;}
+
+.mob_secton_four .mt_60px{ margin-top: 60px !important; }
+
+.mob_secton_four .vodo_mobile_end{ color: #3C3E41; font-family: Roboto Flex;
+font-weight: 400; 
+font-size: 14px; 
+line-height: 24px; 
+text-align: center; margin-top: 20px !important; max-width: 315px; margin: auto;
```

Biome (2.1.2)

[error] Unexpected shorthand property margin after margin-top



(lint/suspicious/noShorthandPropertyOverrides)